sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -137,14 +137,16 @@ def _create_appconfig_pipeline(self, credential, base_url=None, aad_mode=False,
             policies = [
                 self._config.headers_policy,
                 self._config.user_agent_policy,
-                credential_policy,
                 self._config.retry_policy,
+                credential_policy,
                 SyncTokenPolicy(),
                 self._config.logging_policy,  # HTTP request/response log
                 DistributedTracingPolicy(**kwargs),
                 HttpLoggingPolicy(**kwargs)
             ]
 
+        self._policies = policies
```

Comment:
Why we need this?

Reply:
Makes it much easier to grab the policies than digging into the `PipelineClient` object.